testnet/launcher/l1contractdeployer/docker.go

```diff
@@ -74,8 +74,8 @@ func (n *ContractDeployer) RetrieveL1ContractAddresses() (*node.NetworkConfig, e
 	}
 	defer cli.Close()
 
-	// make sure the container has finished execution (3 minutes allows time for L1 transactions to be mined)
-	err = docker.WaitForContainerToFinish(n.containerID, 3*time.Minute)
+	// make sure the container has finished execution (5 minutes allows time for L1 transactions to be mined)
+	err = docker.WaitForContainerToFinish(n.containerID, 5*time.Minute)
```

Comment:
This bump to the timings isn't related to the rest of the PR. We saw it take a bit longer than 3mins in sepolia so the deploy failed, I think it just got longer now we have more L1 contracts to deploy.